app/main/views/frameworks.py

```diff
@@ -3,28 +3,27 @@
 from itertools import chain
 
 from dateutil.parser import parse as date_parse
-from dmapiclient import HTTPError
 from flask import render_template, request, abort, flash, redirect, url_for, current_app, session
 from flask_login import current_user
 import flask_featureflags as feature
-import six
 
 from dmapiclient import APIError
+from dmapiclient import HTTPError
```

Comment:
Can we put this import in the line above?

Reply:
Good spot.  Will do.